### ISSUE TITLE
[Bug] Single comma in JSON response displayed incorrectly

### ISSUE THREAD

**Issue by jaldinger** [getinsomnia/insomnia#277]
## Overview
- `Insomnia Version:` 5.1.1
- `Operating System:` Windows 10 Pro 64-bit
- `Summary: ` A single comma in a JSON response is displayed incorrectly, passing the closing quotation marks onto the next line. Subsequent lines are not highlighted correctly.

## How To Reproduce
1. Create a request 
2. Perform a request which returns a response with a single comma in any value, for example:
{ "ThousandsSeparator" : ",", "DecimalSeparator" : "." }
3. You can see the formatted result is off (new line is inserted after the opening quotation mark and the comma value.
```
{
	"ThousandsSeparator" : ",
",
	"DecimalSeparator" : "."
}
```

## Other Notes
This seems to be caused by the combination of quotation mark and comma, which is interpreted to display with a new line after, when in fact that is not required.

![image](https://cloud.githubusercontent.com/assets/20828844/26691592/b6ba26ae-46cb-11e7-9c67-0d44b71e1055.png)

**Comment by gschier** [comment]
Hey @jaldinger, thanks for reporting this.

I have tried to reproduce this but am unable to do so on Mac or Windows. Maybe there is some weird Unicode characters or something in there that I can't see?

![image](https://cloud.githubusercontent.com/assets/587576/26704513/715d4d66-46e4-11e7-91ad-68e140526588.png)

**Comment by gschier** [comment]
I think this may be fixed in the latest update. I will re-open if it is still an issue.

**Comment by jaldinger** [comment]
Hello Gregory:

I can confirm this is resolved in version 5.3.0. Thank you very much!

Joerg.

**Comment by gschier** [comment]
Woo! Thanks for letting me know @jaldinger
